### DISCUSSION TITLE
Resizing and reordering columns of Roles & Permissions, Presets & Bookmarks and Webhooks

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#11887] [Feature Requests]
and also be ale to show columns added to these system tables.
_In a word, have the same functionalities we already have for users' created tables._

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
